src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/DateTimeFilterIntegration.php

```diff
@@ -12,90 +11,150 @@
  */
 class DateTimeFilterIntegration extends AbstractFilterTestCase
 {
+    /**
+     * @{@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (1 === self::$count || $this->getConfiguration()->isDatabasePurgedForEachTest()) {
+            $this->resetIndex();
+
+            $this->createProduct('foo', []);
+            $this->createProduct('bar', []);
+            $this->createProduct('baz', []);
+        }
+    }
+
+    public function getCreatedProductsDate()
```

Comment:
👀 Drawing your attention here !

Very very very very ugly way to have date with spaced created date, help need please :).

BTW, the tests are green 🎉